config/migrations/20210817155000-delete-signatures.sparql

```diff
@@ -0,0 +1,18 @@
+PREFIX  ext:  <http://mu.semte.ch/vocabularies/ext/>
+PREFIX  public: <http://mu.semte.ch/graphs/public>
+PREFIX  nfo: <http://www.semanticdesktop.org/ontologies/2007/03/22/nfo>
+PREFIX  person: <http://www.w3.org/ns/person>
+
+WITH public:
+DELETE {
+  ?s ?p ?o .
+} WHERE {
+  ?s ext:Handtekening ;
+    ?p ?o .
+};
+
+WITH public:
+DELETE WHERE{ ext:Handtekening  ext:handtekening  nfo:FileDataObject };
+
+WITH public:
+DELETE WHERE{ ext:Handtekening  ext:bevoegdePersoon  person:Person };
```

Comment:
Query werkt idd niet, check dat even met een validator
ik gebruik zelf ook soms https://yasgui.triply.cc/

Daarnaast, als deze 2 deletes zouden werken, deze doen niks.
Dit zijn allebei onderdeel van de eerder verwijderde ?s ?p ?o .
+ je hebt alle `ext:Handtekening` types al verwijdert dus zelfs als er data zou zijn vindt je die op deze manier niet meer

Daarnaast heb je nog de inverse relatie `ext:heeftHandtekening`, maar we kunnen eerst checken of er wel data van bestaat op ACC/PROD, om te checken om de migratie nodig is